ngraph/frontend/paddlepaddle/src/op/squeeze.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (C) 2018-2021 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include <ngraph/opsets/opset6.hpp>
+#include "squeeze.hpp"
+#include <paddlepaddle_frontend/utility.hpp>
+
+namespace ngraph {
+    namespace frontend {
+        namespace pdpd {
+            namespace op {
+
+                NamedOutputs squeeze (const NodeContext& node) {
+                    auto data = node.get_ng_input("X");
+                    auto axes = node.get_attribute<std::vector<int32_t>>("axes");
+                    PDPD_ASSERT(data.get_partial_shape().rank().is_static(), "squeeze: X rank must be static!");
```

Comment:
This is not needed, all validation will be done in Squeeze node

Reply:
@luo-cheng2021 Please remove this

Reply:
fixed in 5db9a7978cf77f1c5a972c114be6be4b693a7c2d